cmd/install.go

```diff
@@ -988,6 +988,24 @@ func installDupeDetection(ctx context.Context, config *configs.Config) (err erro
 	targetDir := filepath.Join(appBaseDir, constants.DupeDetectionSupportFilePath)
 	tmpDir := filepath.Join(targetDir, "temp.zip")
 	for _, url := range constants.DupeDetectionSupportDownloadURL {
+		// Get ddSupportContent and cal checksum
+		ddSupportContent := constants.DupeDetectionSupportContents[path.Base(url)]
+		if len(ddSupportContent) > 0 {
+			ddSupportPath := filepath.Join(targetDir, ddSupportContent)
+			if _, err := os.Stat(ddSupportPath); err == nil {
+				log.WithContext(ctx).Infof("Checking checksum of DupeDetection Support file: %s", ddSupportContent)
+
+				checksum, err := utils.GetChecksum(ctx, ddSupportPath)
```

Comment:
if error - can we consider to download + overwrite?

Reply:
@nguyen202 it will continue to do the next sequence - to download and overwrite as normal.